swap-create@.service

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: MIT
```

Comment:
Maybe create a `units/` subdirectory to store files? I seems likely we'll have more at some point.